<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaperTrackerMainWindow</class>
 <widget class="QWidget" name="PaperTrackerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaperTrack</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>841</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="ImageLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>280</width>
       <height>280</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="SSIDText">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="plainText">
      <string>请输入SSID</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="PasswordText">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="plainText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="BrightnessBar">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>亮度调整</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wifi_send_Button">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FlashFirmwareButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>刷写固件</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="LogText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>821</width>
       <height>171</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>291</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>日志窗口：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IP地址：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restart_Button">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重启</string>
     </property>
    </widget>
    <widget class="QLabel" name="SerialConnectLabel">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>串口未连接</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="WifiConnectLabel">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>250</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>wifi未连接</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>旋转角度调整</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="RotateImageBar">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="UseFilterBox">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>启用滤波</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>500</width>
       <height>441</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>441</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>639</width>
        <height>500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>500</height>
       </size>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>cheekPuffLeft</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mouthLeft</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tongueUp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>cheekPuffRight</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tongueLeft</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>jawOpen</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mouthRight</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>jawRight</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tongueDown</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>jawLeft</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tongueOut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tongueRight</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="CheekPullLeftValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="CheekPullRightValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="JawLeftValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>120</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="JawOpenValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>80</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="MouthLeftValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>200</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="JawRightValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>160</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="TongueOutValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>280</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="MouthRightValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>240</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="TongueDownValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>360</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="TongueUpValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>320</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="TongueRightValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>440</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="TongueLeftValue">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>400</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QScrollBar" name="CheekPuffLeftBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="CheekPuffRightBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="JawOpenBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="JawLeftBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="MouthLeftBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="JawRightBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="TongueOutBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="MouthRightBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="TongueDownBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>370</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="TongueUpBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="TongueRightBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>450</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="TongueLeftBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>410</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="StartCalibrationButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始标定(开发中)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CalibrationModeComboBox">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Neutral模式</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Full模式</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ImageLabelCal">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>190</y>
       <width>280</width>
       <height>280</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="MainPageButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>主页面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalibrationPageButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>标定页面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
